.github/workflows/release.yaml

```diff
@@ -14,7 +19,7 @@ jobs:
   validation:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
The changes here address validation issues
<img width="1291" alt="Screen Shot 2023-01-05 at 7 01 37 AM" src="https://user-images.githubusercontent.com/26427366/210810950-2b8471e2-a595-4c84-8b38-d4052a3fbcca.png">